### DISCUSSION TITLE
Unable to Upload from ant device Web Mobile Device

### DISCUSSION THREAD

**Discussion by thorrrr** [photoprism/photoprism#2974] [Q&A]
Unable to Upload from any device Web Mobile Device  i have it in a stack in Portianer.
I have it conected to another server which it sees and has dragged in some images.
But anythime i use the Upload Feature it goes through the motion images get uploaded but the folder is always blank.
LOG see [here](https://logs.notifiarr.com/?39c3187557b5e826#9PR1aN2gCQEJKDiKLjA6NbSY7XNdmDEL4599JhJZM9Ht)

**Comment by graciousgrey** [comment]
Make sure you have import AND originals folder configured with the right permissions.
Also make sure that the import folder is not located within originals.
